src/styles/mo-components/_buttons.scss

```diff
@@ -1,7 +1,17 @@
 button {
   &.btn {
+    font-size: 1em;
+    padding: .375rem .75rem;
+
+    &--large {
+      padding: .5rem 1rem;
+      font-size: 1.5em;
+      height: auto;
+    }
+
     &.azure {
       @include btn--azure;
+      color: $white;
```

Comment:
The btn--azure mixing should take care of the text color - otherwise the .white color utility class should. Is it getting overridden somehow?

Reply:
yeah, idk why I had it